### DISCUSSION TITLE
version of data published on http://coastsat.wrl.unsw.edu.au/

### DISCUSSION THREAD

**Discussion by weiguochina** [kvos/CoastSat#503] [Q&A] [answered]
It is exciting to see many new updates and versions of the package have been published. I am curious about the visualization and data on http://coastsat.wrl.unsw.edu.au/. Which version do they use?

**Comment by kvos** [comment]
CoastSat 1.X was used to generate those time-series.

**Comment by kvos** [comment, marked as answer]
for the latest data, running until Jan 2025, please go to http://coastsat.space
